driver/flambda_backend_flags.ml

```diff
@@ -219,18 +207,13 @@ module Flambda2 = struct
   let o2_flags () =
     cse_depth := 2;
     join_depth := 2;
```

Comment:
Could you take this opportunity to remove the `join_depth` line here ? I forgot to do it before we merged the corresponding PR.

Reply:
Ah yes, well spotted.  Done.